monai/bundle/meta_schema.json

```diff
@@ -0,0 +1,220 @@
+                        "^.+$": {
+                            "$ref": "#/$defs/argument"
+                        }
+                    }
+                },
+                "outputs": {
+                    "type": "object",
+                    "patternProperties": {
+                        "^.+$": {
+                            "$ref": "#/$defs/result"
+                        }
+                    }
+                },
+                "post_processed_outputs": {
+                    "$comment": "Return value format after post-processing, not needed if not changed.",
+                    "type": "object",
+                    "patternProperties": {
+                        "^.+$": {
+                            "$ref": "#/$defs/result"
+                        }
+                    }
+                }
+            },
+            "required": [
+                "inputs",
+                "outputs"
+            ]
+        }
+    },
+    "type": "object",
+    "properties": {
+        "schema": {
+            "description": "URL of the schema file.",
+            "type": "string"
+        },
+        "version": {
+            "description": "Version number of the bundle.",
+            "type": "string"
+        },
+        "changelog": {
+            "description": "Dictionary relating previous version names to strings describing the version.",
+            "type": "object"
+        },
+        "monai_version": {
+            "description": "Version of MONAI the bundle was generated with.",
+            "type": "string"
+        },
+        "pytorch_version": {
+            "description": "Version of PyTorch the bundle was generated with.",
+            "type": "string"
+        },
+        "numpy_version": {
+            "description": "Version of NumPy the bundlewas generated with.",
+            "type": "string"
+        },
+        "required_packages_version": {
+            "description": "Dictionary relating required package names to their versions. The bundle requires these packages to operate which are additional to the base requirements for MONAI.",
+            "type": "object"
+        },
+        "task": {
+            "description": "Plain-language description of what the bundle is meant to do.",
+            "type": "string"
+        },
+        "description": {
+            "description": "Longer form description of what the bundle is, what it does, etc.",
+            "type": "string"
+        },
+        "authors": {
+            "description": "State author(s) of the bundle.",
+            "type": "string"
+        },
+        "copyright": {
+            "description": "State copyright of the bundle.",
+            "type": "string"
+        },
+        "data_source": {
+            "description": "Where to download or prepare the data used in this bundle.",
+            "type": "string"
+        },
+        "data_type": {
+            "description": "Type of the data, like: `dicom`, `nibabel`, etc.",
```

Comment:
To resolve ambiguity of either user should put `nibabel` vs. `nii`, would it be helpful to say?
```
"Type of the data, like: `dicom`, `nibabel`, etc. (e.g., 'dicom', 'nii', 'nii.gz')",
```

Reply:
Sounds good